src/api/app/models/staging/staged_requests.rb

```diff
@@ -120,7 +125,7 @@ def link_package(bs_request_action)
 
     package_rev = backend_package_information['srcmd5']
 
-    link_package = Package.find_or_create_by!(project: staging_project, name: bs_request_action.target_package)
```

Comment:
We originally had the find_or_create to make superseding work - but as requests don't supersede each other, Eduardo and I discussed that errors are easier to handle from the client's side.